### DISCUSSION TITLE
Task Tracking for v3

### DISCUSSION THREAD

**Discussion by dscham** [NginxProxyManager/nginx-proxy-manager#4195] [Q&A]
Is there a board with the tasks at hand for v3? I may contribute, but I don't really know where to start. Half of the projects code looks like it's placeholder right now.

I get the ideas, but I don't want to step on anyones feet. Or maybe I'm even seeing something wrong.

**Comment by Revan335** [comment]
Its look like deleted the old postings about V3 that I saw in the past.
